packages/manager/modules/pci/src/projects/project/notebooks/components/configuration-command/configuration-command.controller.js

```diff
@@ -17,8 +18,32 @@ export default class {
     this.getConfigurationCommand();
   }
 
+  getFilteredVolumes() {
+    return this.notebookSpecs.volumes.flatMap((volume) => {
+      let toRemove = false;
+      let internalKey;
+      Object.keys(volume).forEach((key) => {
```

Comment:
I wonder if `_.filter`. or `_.find` can help simplify logic here

Reply:
Hi @marie-j , as seen together, using `_.filter` or `_.find` does not seem to simplify, as we'd need to map the result of filter and loop again through all properties to achieve the same result.